src/web_ui/src/apis/products.js

```diff
@@ -0,0 +1,102 @@
+export const products = [
+  {
+    "id": 1,
+    "category": "VEGETABLES",
+    "name": "jackfruit",
+    "created_at": "2012-04-23T18:25:43.511Z",
+    "updated_at": "2012-04-23T18:25:43.511Z",
+    "image": "https://loremflickr.com/320/240/dog",
+    "time_ago": "1 hour ago",
+    "farm_name": "Fastfarm"
+  },
+  {
+    "id": 2,
+    "category": "FRUITS",
+    "name": "apples",
+    "created_at": "2012-04-23T18:25:43.511Z",
+    "updated_at": "2012-04-23T18:25:43.511Z",
+    "image": "https://loremflickr.com/320/240/paris",
+    "time_ago": "2 minutes ago",
+    "farm_name": "Dairyfarm"
+  },
+  {
+    "id": 3,
+    "category": "FRUITS",
+    "name": "apples",
+    "created_at": "2012-04-23T18:25:43.511Z",
+    "updated_at": "2012-04-23T18:25:43.511Z",
+    "image": "https://loremflickr.com/320/240/brazil",
+    "time_ago": "2 hours ago",
+    "farm_name": "Dairyfarm"
+  },
+  {
+    "id": 4,
+    "category": "SALADS",
+    "name": "lemon",
+    "created_at": "2012-04-23T18:25:43.511Z",
+    "updated_at": "2012-04-23T18:25:43.511Z",
+    "image": "https://loremflickr.com/320/240?random",
+    "time_ago": "24 hours ago",
+    "farm_name": "Fastfarm"
```

Comment:
Refer to overall comment on PR on which fields we require. Not sure about farm_name in this case.

Reply:
This should be a farm id but this can work for now as the mock object